python/GafferDispatchUI/LocalDispatcherUI.py

```diff
@@ -240,7 +240,8 @@ def __init__( self, jobPool, **kw ) :
 								self.__detailsDirectory.setTextSelectable( True )
 
 					with GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Vertical, spacing=10, borderWidth=10, parenting = { "label"  : "Messages" } ) as self.__messagesTab :
-						self.__messageWidget = GafferUI.MessageWidget()
+						self.__messageWidget = GafferUI.MessageWidget( toolbars = True, follow = True )
```

Comment:
Nice! As a follow up, it would be great to look at `_LocalJobsWindow.__updateMessages()`. It's currently doing some crazy clear-and-resend-all-the-messages thing. We can talk about that after though...